readme.md

```diff
@@ -56,6 +56,7 @@ Configuration:
   --print-config                 Print the normalized config for the given config and options, then exit.                                  [boolean]
   --additional-trace-categories  Additional categories to capture with the trace (comma-delimited).
   --config-path                  The path to the config JSON.
+                                 An example config file: lighthouse-core/config/lr-desktop-config.js
```

Comment:
this section mirrors the printout from `--help`, so will need to make these changes in `lighthouse-cli/cli-flags.js` to have them there as well